Assets/Plugins/Source/Editor/Utility/UnityPackageCreationUtility.cs

```diff
@@ -116,29 +116,6 @@ public static async Task CreatePackage(PackageType packageType, IProgress<Create
             ValidatePackage(packagingConfig.pathToOutput);
         }
 
-        private static void CreateDotUnityPackage(string outputPath, string json_file,
```

Comment:
Why was this deleted?

Reply:
For a few reasons. The json file that we have for it is wildly outdated, and we don't make the option to create a `.unitypackage` available anymore in the UI. The short answer is that it was removed because it hasn't worked for a while now.